src/components/Image.js

```diff
@@ -1,32 +1,18 @@
 import React from 'react';
-import { useStaticQuery, graphql } from 'gatsby';
-import Img from 'gatsby-image';
+import PropTypes from 'prop-types';
 
-/*
- * This component is built using `gatsby-image` to automatically serve optimized
- * images with lazy loading and reduced file sizes. The image is loaded using a
- * `useStaticQuery`, which allows us to load the image from directly within this
- * component, rather than having to pass the image data down from pages.
- *
- * For more information, see the docs:
- * - `gatsby-image`: https://gatsby.dev/gatsby-image
- * - `useStaticQuery`: https://www.gatsbyjs.org/docs/use-static-query/
- */
+const Image = ({ src, alt, width }) => {
+  return <img src={src} alt={alt} width={width} />;
```

Comment:
```suggestion
const Image = (props) => {
  return <img {...props} />;
```

Reply:
when I try to do this, i get a linting error because the linter wants there to be an alt prop, so I think i'm just going to keep it in the structure that I currently have it in

Reply:
@caylahamann You can get around the lint error and apply @zstix's suggestion by destructuring the `alt` prop and spreading the rest of them like this:

```js
const Image ({ alt, ...props }) => (
  <img alt={alt} {...props} />
)
```